plugin/core/main.py

```diff
@@ -167,7 +167,7 @@ def handle_initialize_result(result, client, window, project_path, config):
     client.send_notification(Notification.initialized())
     if config.settings:
         configParams = {
-            'settings': config.settings
+            'settings': config.get_settings(window)
```

Comment:
This get-settings-by-window function looks incomplete (and perhaps not needed), correct?